clip_retrieval/clip_inference/worker.py

```diff
@@ -46,8 +46,8 @@ def worker(
         input_dataset = braceexpand(input_dataset)
 
     def reader_builder(sampler):
-        _, preprocess = load_clip(
-            clip_model=clip_model, use_jit=use_jit, warmup_batch_size=batch_size, clip_cache_path=clip_cache_path
+        _, preprocess = load_clip_without_warmup(
```

Comment:
This is not needed. The caching already makes it so load + warmup is done only once

Reply:
I was seeing two print-statements of the model "warming up" and "done" so i thought it may be necessary. if they are superfluous print statements we can revert

Reply:
If that is happening, that means the cache is not working, it should be fixed

Reply:
fixed